lib/rules/web/admin_console/4.5/catalog.xyaml

```diff
@@ -129,13 +129,17 @@ click_subscribe_button:
   params:
     button_text: Subscribe
   action: click_button_text
+click_install_button:
+  params:
+    button_text: Install
+  action: click_button_text
 subscribe_using_default_params:
   action: wait_box_loaded
   action: click_subscribe_button
 set_custom_channel_and_subscribe:
   action: wait_box_loaded
   action: choose_update_channel
-  action: click_subscribe_button
+  action: click_install_button
```

Comment:
in 4.5 `click_subscribe_button` just click on a button with text `Install`

```
click_subscribe_button:
  params:
    button_text: Install
  action: click_button_text
```

Reply:
@yapei hmm...in my branch that's not defined.  I guess it must've been udpated before I started this.

Reply:
@yapei, yeah that's the case, my orignal branch didn't have that.  Thanks for pointing it out, I'll revet that change
```
click_subscribe_button:
  params:
    button_text: Subscribe
  action: click_button_text
```